laghos_rom.hpp

```diff
@@ -65,6 +66,7 @@ struct ROM_Options
     bool GramSchmidt = false; // whether to use Gram-Schmidt with respect to mass matrices
     bool RK2AvgSolver = false; // true if RK2Avg solver is used for time integration
     bool paramOffset = false; // used for determining offset options in the online stage, depending on parametric ROM or non-parametric
+    int offsetType = 0; // 0 -> save and load offset; 1 -> use initial state as offset; 2 -> use solution from previous window as offset
```

Comment:
Is it possible to switch to an int enum for readability, instead of 0, 1, and 2?

Reply:
@kevinhkhuynh Thanks Kevin. I will look into this switch.

Reply:
I agree, an enum would make the code clearer.